app/utils.py

```diff
@@ -200,3 +201,24 @@ def log_user(user, message):
         message,
         extra=user,
     )
+
+
+def get_ip_address():
+    ip = None
+    if current_app.config["HOST"] == "local":
+        ip = request.remote_addr
+    elif current_app.config["HOST"] == "hosted":
+        ip = request.headers.get("X-Forwarded-For")
+    elif current_app.config["HOST"] == "test":
```

Comment:
Rather than this, why not change `check_ip_should_be_throttled` and then simply return false if the current `HOST` is not test?

Reply:
I've removed references to checking HOST and instead mock the response of get_ip_address in the tests, for tests.